RoslynPluginGenerator/CommandLine/CmdLineResources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="ArgDesciption_CustomNuGetRepo" xml:space="preserve">
     <value>/customnugetrepo:[Path/Url to Repo] - overwrites default nuget behaviour and only uses this repository.</value>
   </data>
+  <data name="ArgDesciption_Language" xml:space="preserve">
+    <value>/language:[cs|vb] - specify analyzer languages. Default: cs</value>
```

Comment:
With the help message it would be nicer with something like this:
```suggestion
    <value>&lt;language&gt; - specify analyzer languages. Possible values: cs, vb. Default: cs</value>
```